test/browser/test_sdl2_timer.c

```diff
@@ -10,13 +10,18 @@
 #include <emscripten.h>
 #include <SDL/SDL.h>
 
-Uint32 SDLCALL report_result(Uint32 interval, void *param) {
+Uint32 start_time;
+
+Uint32 SDLCALL timer_callback(Uint32 interval, void *param) {
   static int reported = 0;
-  if (reported) return 0;
+  assert(!reported);
   reported = 1;
-  SDL_Quit();
   int result = *(int *)param;
-  printf("report_result: %p %d\n", param, result);
+  printf("timer_callback: %p %d\n", param, result);
+  assert(result == 5);
+  Uint32 now = SDL_GetTicks();
+  assert(now > start_time + 500);
```

Comment:
```suggestion
  assert(now >= start_time + 500);
```
Might avoid some type of rare flake.